front/src/components/RecipeSelection/RecipeSelection.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import SelectedRecipeList from 'components/SelectedRecipeList/SelectedRecipeList';
+import DropdownSearch from 'components/UI/Dropdown/DropdownSearch/DropdownSearch';
+import getIngredients from 'helpers/getIngredients';
+import useAsync from 'hooks/useAsync';
+import style from './RecipeSelection.module.scss';
```

Comment:
```suggestion
import styles from './RecipeSelection.module.scss';
```

[Стайлгайд](https://github.com/Prostoe-menu/prostoemenu_dev/blob/frontend/front/src/docs/style-guide.md#%D0%BF%D0%BE%D1%80%D1%8F%D0%B4%D0%BE%D0%BA-%D0%B8%D0%BC%D0%BF%D0%BE%D1%80%D1%82%D0%BE%D0%B2)